.github/workflows/test.yml

```diff
@@ -346,6 +346,12 @@ jobs:
       - name: Check cargo deny advisories/licenses
         if: needs.changes.outputs.dependencies == 'true' || needs.changes.outputs.deny == 'true'
         run: make check-deny
+      - name: Install the 3rd-party license tool
+        if: needs.changes.outputs.dependencies == 'true' || needs.changes.outputs.deny == 'true'
```

Comment:
```suggestion
        if: needs.changes.outputs.dependencies == 'true'
```

Doesn't seem like you'd need to run it on deny changes.

Reply:
Ah, no, I just copied from the previous line, fixed in 841659fb8.